apps/mobile/src/app/wallet/settings/networks/index.tsx

```diff
@@ -28,23 +38,26 @@ export default function SettingsNetworksScreen() {
         }}
       >
         <Cell
-          isSelected
+          isSelected={settings.network.id === 'mainnet'}
+          onPress={() => onPressChangeNetwork('mainnet')}
           title={t`Mainnet`}
-          subtitle={t`Enabled`}
+          subtitle={settings.network.id === 'mainnet' ? t`Enabled` : t`Disabled`}
```

Comment:
It seems like we could clean this up a bit but I think it's fine as is. 

Maybe destructure and renaming `settings.network.id` earlier? 

```
const { network: { id: networkId } } = settings
```

Or we could have a `switch` to pass props to `<Cell` based on `settings.network.id`

Reply:
Just a style preference, but isn't accessing a few properties deep of an object cleaner than a nested destructure?

On grounds that `{ network: { id: networkId } }` is more syntax heavy than `settings.network.id`?

Reply:
I think it's cleaner to do one destructure to get what you need than multiple chains. Just my preference though